### PR TITLE
306948: Added STORAGEACCOUNT_PREFIX variable in all infra patch files

### PR DESCRIPTION
**Pull request by ganeshg2491**
# **What this PR does / why we need it**:
Added STORAGEACCOUNT_PREFIX variable.
Relates to ADO Work Item [AB#306948](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/306948)

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
gif

### CODE REVIEW

File: services/eutd/eutd-trade-exports-ai/trade-exports-packinglistai-ui/infra/snd/01/patch.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ spec:
     azrMSTPrivateLinkDNSSubscriptionID: ${AZR_MST_PRIVATELINKDNS_SUBSCRIPTION_ID}
     azrMSTPrivateLinkDNSUKSouthResourceGroupName: ${AZR_MST_PRIVATELINKDNS_UKSOUTH_RG}
     azrMSTPrivateLinkDNSUKWestResourceGroupName: ${AZR_MST_PRIVATELINKDNS_UKWEST_RG}
+    storageAccountPrefix: ${STORAGEACCOUNT_PREFIX}
```
[the remark this answers is not in the document]
Comment (ganeshg2491) [reply services/eutd/eutd-trade-exports-ai/trade-exports-packinglistai-ui/infra/snd/01/patch.yaml:31]: Which requirement @DaRoszko ? I didn't get this. This PR is just to add platform variables to all infra helm chart patch files.

[the remark this answers is not in the document]
Comment (DaRoszko) [reply services/eutd/eutd-trade-exports-ai/trade-exports-packinglistai-ui/infra/snd/01/patch.yaml:31]: when we scaffold flux services, in this in the scaffolder for flux? or is this separate?

[the remark this answers is not in the document]
Comment (DaRoszko) [reply services/eutd/eutd-trade-exports-ai/trade-exports-packinglistai-ui/infra/snd/01/patch.yaml:31]: Everytime we scaffold, we need to include all stuff @kbabigumira  @aadev1  ?

[the remark this answers is not in the document]
Comment (aadev1) [reply services/eutd/eutd-trade-exports-ai/trade-exports-packinglistai-ui/infra/snd/01/patch.yaml:31]: @DaRoszko - No we don't need to include all stuff.  The scaffolder will only create files for the team being onboarded.
Here's an example PR when Ken onboarded ffc-find.

https://github.com/DEFRA/adp-flux-services/pull/54/files

For this PR Ganesh has added his changes to the scaffolder script and then used the script locally to generate all the Helm chart patch files with his changes required by the already Onboarded teams.  

It's something we can probably get away with at the moment when we make Platform updates to Flux services, but it's something we need to think about when we make changes in the future that need to be added to existing Team files in the Flux services repository.

**Comment by aadev1** [comment]
@kbabigumira @DaRoszko - Do we also need to add these updates to the new Teams Onboarded since this PR was raised?  coreai-mcu and another one I think.

**Review by aadev1 — APPROVED** [review]

**Review by DaRoszko — APPROVED** [review]

**Pull request merged**